SourceS/file_util.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#if _POSIX_C_SOURCE >= 200809L
+#include <unistd.h>
+#else
+#include <cstdio>
+#endif
+
+namespace dvl {
+
+inline bool FileExists(const char *path)
+{
+#if _POSIX_C_SOURCE >= 200809L

```

Comment:
I think access is already in 200112L
```suggestion
#if _POSIX_C_SOURCE >= 200112L

```

Reply:
Done